creator-node/nginx_conf/nginx.conf

```diff
@@ -46,10 +47,12 @@ http {
             proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
             proxy_cache_background_update on;
 
-            # Cache only responses with status code = 200 for some duration before considered stale.
+            # Cache responses with certain status codes for some duration before considered stale.
             # Stale implies content will be fetched from the upstream server. Stale content WILL NOT
             # BE REMOVED from the cache.
             proxy_cache_valid 200 12h;
+            proxy_cache_valid 403 1h;
```

Comment:
caching the 403 to 1hr will return 403 for everyone right?
if so, then this causes issues for premium content when signature verification fails for a user but should succeed for another one. the latter user will also incorrectly get 403

Reply:
removed